src/components/Menu/index.web.tsx

```diff
@@ -135,10 +134,22 @@ export function Trigger({children, label}: TriggerProps) {
             },
             props: {
               ...props,
-              // disable on web, use `onPress`
-              onPointerDown: () => false,
```

Comment:
The `=> false` part was not doing anything here. React does not take return value of an event handler into account.

The reason this worked before is because this `onPointerDown` overrode `props.onPointerDown` due to spread. I replaced it with `undefined` to make that clearer.

Note that in the other implementation (pre-ALF), there is a `e.preventDefault` call. Shouldn't we add it here? The answer is no. We do not need to replicate it here. The only reason it existed in the non-ALF implementation is because we didn't have control over how props are spread (we used `asChild` there). So we called `e.preventDefault` _not even for the browser's sake_ but because Radix itself checks `e.defaultPrevented` before running its own logic. Since we have control over props in this version, we don't need `e.preventDefault` — and that's why it worked without it. (Again, `=> false` didn't do anything.)

Reply:
Thanks for the comment, much clearer

Reply:
np, this was a solid nerdsnipe — one had me wondering “but what does it do?” and the second one was “but how does it work?” 😅